rchain_grpc/rho_types.py

```diff
@@ -3,19 +3,22 @@
 from typing import Any, List, Tuple, TypeVar, Union
 
 import toolz
-from google.protobuf.empty_pb2 import Empty
 from google.protobuf.message import Message
 
 from ._grpc_containers import (RepeatedCompositeFieldContainer,
                                RepeatedScalarFieldContainer)
-from .generated.CasperMessage_pb2 import DataWithBlockInfo
-from .generated.RhoTypes_pb2 import Channel, Expr, Par, Var
+
+# TODO: please check, but without it doesn't work
+from google.protobuf.pyext._message import \
+    RepeatedCompositeContainer, RepeatedScalarContainer
+
+from .generated.RhoTypes_pb2 import Expr, Par
+from .generated.CasperMessage_pb2 import DataAtNameQuery
 
 
 def e_map_body_to_dict(body):
-    kvs = body['kvs']
     ret = {}
-    for kv in kvs:
+    for kv in body.get('kvs'):
```

Comment:
i prefer `[]` here because it will generate an error when they change something in response.

Reply:
you mean add `[]` as default to get?